src/resources/js/ticket-details.js

```diff
@@ -0,0 +1,60 @@
+var tribe_ticket_details = tribe_ticket_details || {};
+
+( function( obj ) {
+	'use strict';
+
+	obj.init = function( detailsElems ) {
+		obj.event_listeners();
+	}
+
+	obj.event_listeners = function() {
+		// Add keyboard support for enter key.
+		document.addEventListener( 'keyup', function( event ) {
+			// Toggle open like click does.
+			if ( 13 === event.keyCode ) {
+				obj.toggle_open( event.target );
+			}
+		} );
+
+		document.addEventListener( 'click', function( event ) {
```

Comment:
curious why you're not using jQuery in this file, while all the other JS uses jQuery, it would make event handling much easier (especially `obj.toggle_open`).